app/src/main/java/org/oppia/app/customview/interaction/FractionInputInteractionView.kt

```diff
@@ -26,11 +31,19 @@ class FractionInputInteractionView @JvmOverloads constructor(
     hintText = hint.toString()
   }
 
-  override fun onFocusChange(v: View?, hasFocus: Boolean) = if (hasFocus) {
+  override fun onFocusChange(v: View, hasFocus: Boolean) = if (hasFocus) {
     hint = ""
     typeface = Typeface.DEFAULT
+    showSoftKeyboard(v, context)
   } else {
     hint = hintText
-    setTypeface(typeface, Typeface.ITALIC)
+    if (text.isEmpty()) setTypeface(typeface, Typeface.ITALIC)
+    hideSoftKeyboard(v, context)
+  }
+
+  override fun onKeyPreIme(key_code: Int, event: KeyEvent): Boolean {
```

Comment:
Should be ``keyCode`` since we use lowerCamelCase for identifiers. Ditto for other files changed in this PR (I counted 2 more occurrences).

Reply:
Done in all three input view classes